tests_end_to_end/xsiam/test_e2e_demisto_sdk_flow_playbook_xsiam.py

```diff
@@ -0,0 +1,198 @@
+    e2e_tests_utils.cli(f"mkdir {tmpdir}/git")
+    e2e_tests_utils.git_clone_demisto_sdk(
+        destination_folder=f"{tmpdir}/git/demisto-sdk",
+        sdk_git_branch=DEMISTO_GIT_PRIMARY_BRANCH,
+    )
+    
+    repo = Repo(tmpdir)
+    pack, pack_name, source_pack_path = e2e_tests_utils.create_pack(repo)
+    playbook, playbook_name, source_playbook_path = e2e_tests_utils.create_playbook(pack, pack_name)
+    assert Path(source_playbook_path).exists()
+
+    logger.info(f"Trying to upload pack from {source_pack_path}")
+    Uploader(input=source_pack_path, insecure=True, zip=True, marketplace=MarketplaceVersions.MarketplaceV2).upload()
+    
+    # Preparing updated pack folder
+    e2e_tests_utils.cli(f"mkdir {tmpdir}/Packs/{pack_name}_testsuite")
+
+    logger.info(
+        f"Trying to download the updated playbook from {playbook_name} to {tmpdir}/Packs/{pack_name}_testsuite/Playbooks"
+    )
+    Downloader(
+        output=f"{tmpdir}/Packs/{pack_name}_testsuite",
+        input=(playbook_name,),
+        insecure=True,
+        system=True,
+        item_type='Playbook'
+    ).download()
+    dest_playbook_path = Path(
+        f"{tmpdir}/Packs/{pack_name}_testsuite/Playbooks/{playbook_name}.yml"
+    )
+    assert not dest_playbook_path.exists()
+
+    logger.info(
+        f"Generating docs (creating a readme file) for playbook {source_playbook_path}"
+    )
+    generate_playbook_doc.generate_playbook_doc(input_path=str(source_playbook_path))
+    assert Path(
+        f"{tmpdir}/Packs/{pack_name}/Playbooks/{playbook_name}_README.md"
+    ).exists()
+
+    logger.info(f"Formating playbook {source_playbook_path}")
+    with ChangeCWD(pack.repo_path):
+        format_manager(
+            input=str(source_playbook_path),
+            assume_answer=True,
+        )
+        logger.info(f"Validating playbook {source_playbook_path}")
+        OldValidateManager(file_path=str(source_playbook_path)).run_validation()
+
+        logger.info(f"Uploading updated playbook {source_playbook_path}")
+        Uploader(input=source_pack_path, insecure=True, zip=True, marketplace=MarketplaceVersions.MarketplaceV2).upload()
+
+
+def test_e2e_demisto_sdk_flow_playbook_client(tmpdir, verify_ssl: bool = False):
+    """This flow checks:
+        1. Creates a new playbook and uploading it to the machine using an http request.
+        2. Downloads the playbook using demisto-sdk download command.
+        3. Downloads the script CommonServerUserPowerShell using demisto-sdk upload command.
+        4. Generates docs for the playbook using demisto-sdk generate-docs command.
+        5. Formatting the playbook using the demisto-sdk format command.
+        6. Validates the playbook using the demisto-sdk validate command.
+        7. Uploads the playbook using the demisto-sdk upload command.
+        8. Deletes the playbook using an http request.
+    """
+    demisto_client = get_client_from_server_type(verify_ssl=verify_ssl)
```

Comment:
```suggestion
    demisto_client = get_client_from_config(verify_ssl=verify_ssl)
```

I suggest to use `get_client_from_config` because you don't need to find the correct client dynamiclly, you know here you need xsiam.

also use it from a fixture, see example here: https://github.com/demisto/content/blob/master/Tests/tests_e2e/conftest.py#L49